### PR TITLE
[UTOPIA-795]add migration to update ministry name

### PR DESCRIPTION
**Pull request by binggaofw**
# Description

This PR includes the following proposed change(s):

- add new migration to update existing pia-intake form ministry name(if any)  to new ministry name 

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation
- [ ] Version change

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Development Dependency Working Agreement
- [x] My code DOES NOT include the importing of new dependencies into the DPIA ecosystem
- [ ] My code DOES include the importing of new dependencies into the DPIA ecosystem
**If new dependencies are being introduced to the DPIA ecosystem:**
- [ ] The functionality of the dependency drastically reduces code complexity and makes my changes more easily maintainable and readible 
- [ ] The dependency being introduced does not contain multiple layers of nested dependencies introducing maintainability complexity to the DPIA ecosystem

## Frontend Development Changes
- [x] N/A
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to project documentation or diagrams that reflect my changes
- [ ] New and existing unit tests pass locally with my changes
- [ ] My code follows Airbnb React Style Guidelines

## API Development Changes
- [] N/A
- [x] I have performed a self-review of my own code
- [x] My code follows standards and practices outlined in the BC Government API Development Guidelines
- [x] New and existing unit tests pass locally with my changes
- [ ] My changes includes Swagger documentation updates that reflect the changes I am introducing

## Definition of Done

Definition of Done

### CODE REVIEW

**Comment by Bottle7** [comment]
I think you're missing housing.

**Comment by binggaofw** [comment]
the housing and emergency management are new ministry, so there is no existing pia link to these new added ministries, so we do not need to run migration for them @Bottle7

**Review by Bottle7 — DISMISSED** [review]

File: src/backend/src/migrations/1678996557122-update-ministry-name.ts [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+    await queryRunner.query(`UPDATE "pia-intake"  SET "ministry" ='WATER_LAND_AND_RESOURCE_STEWARDSHIP'
+        WHERE ministry='LAND_WATER_AND_RESOURCE_STEWARDSHIP'`);
+
+    await queryRunner.query(`UPDATE "pia-intake"  SET "ministry" ='PUBLIC_SAFETY_AND_SOLICITOR_GENERAL'
+        WHERE ministry='PUBLIC_SAFETY_AND_SOLICITOR_GENERAL_AND_EMERGENCY_BC'`);
+  }
+
+  public async down(): Promise<void> {
+    // no going back
```
[the remark this answers is not in the document]
Comment (binggaofw) [reply src/backend/src/migrations/1678996557122-update-ministry-name.ts]: done

File: src/backend/src/migrations/1678996557122-update-ministry-name.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+    await queryRunner.query(`UPDATE "pia-intake"  SET "ministry" ='JOBS_ECONOMIC_RECOVERY_AND_INNOVATION'
+        WHERE ministry='JOBS_ECONOMIC_DEVELOPMENT_AND_INNOVATION'`);
+
+    await queryRunner.query(`UPDATE "pia-intake"  SET "ministry" ='LAND_WATER_AND_RESOURCE_STEWARDSHIP'
+        WHERE ministry='WATER_LAND_AND_RESOURCE_STEWARDSHIP'`);
+
+    await queryRunner.query(`UPDATE "pia-intake"  SET "ministry" ='PUBLIC_SAFETY_AND_SOLICITOR_GENERAL_AND_EMERGENCY_BC'
+        WHERE ministry='PUBLIC_SAFETY_AND_SOLICITOR_GENERAL'`);
+  }
```
Comment (kushal-arora-fw) [inline src/backend/src/migrations/1678996557122-update-ministry-name.ts:32]: down migrations are usually reverse order. Can you please reverse the order of these calls.

Comment (binggaofw) [reply src/backend/src/migrations/1678996557122-update-ministry-name.ts:32]: good catch, will do now

Comment (binggaofw) [reply src/backend/src/migrations/1678996557122-update-ministry-name.ts]: done

**Review by kushal-arora-fw — APPROVED** [review]

**Pull request merged**